### PR TITLE
Fixed #141: Modified design for lighbox with some more design features

### LINKED ISSUE #141: Unmodified links in the lightbox.
All the feeds with links particularly in the last are replaced by loklak.org links as returned from Api Response(which redirect to the same link)  in the feed lighbox. Moreover , the "View on Twitter" is not working.
screen shot 2016-12-23 at 3 50 54 am
screen shot 2016-12-23 at 3 52 39 am

### PR DESCRIPTION
**Pull request by Achint08**
gh-pages :- https://achint08.github.io/loklak_search/

### CODE REVIEW

**Comment by Achint08** [comment]
screen shot 2016-12-24 at 3 40 01 am
Screenshot of the new design feature.

**Comment by Achint08** [comment]
I changed the API request link because previously there was an error shown when I deployed gh-pages .
screen shot 2016-12-24 at 3 53 57 am

This has been fixed now.

File: src/app/shared/services/search/search.service.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { ApiResponse } from '../../../models/api-response';
 
 @Injectable()
 export class SearchService {
-	private static apiUrl: URL = new URL('http://api.loklak.org/api/search.json');
+	private static apiUrl: URL = new URL('https://api.loklak.org/api/search.json');
```
Comment (hemantjadon) [inline src/app/shared/services/search/search.service.ts]: Please remove `https` and use `http` only as the backend is not on `https`

File: src/app/feed/feed-card/feed-card.component.scss [hunk 2]
```diff
@@ -171,6 +180,48 @@ a.post-link {
...
+				}
+            
+            .lightbox-space {
+                margin-left: 60px;
+            }
+        }
 	}
 
-}
+}
```
Comment (hemantjadon) [inline src/app/feed/feed-card/feed-card.component.scss]: Insert final newline

**Comment by Achint08** [comment]
@hemantjadon I was previously having the above mentioned error in the previous comment. "Blocked loading mixed active content" for my gh-pages if I used http instead for https. Any suggestion?

**Comment by hemantjadon** [comment]
@Achint08 It is not an error as your server is running on `https` and api runs on `http`. You just have to enable unsafe scripts in the browser. In chrome there it appears as a shield in right end of URL bar. This will temporarily allow http connection on your https client.

**Comment by mariobehling** [comment]
@hemantjadon May be a bit out of scope for this PR, but now that GitHub supports https, how can we make the switch to https?

**Comment by mariobehling** [comment]
I opened an issue here https://github.com/fossasia/loklak_search/issues/146

File: src/app/shared/services/search/search.service.ts (hunk 1 above)
Comment (Achint08) [inline src/app/shared/services/search/search.service.ts]: Done. (Y)

File: src/app/feed/feed-card/feed-card.component.scss (hunk 2 above)
Comment (Achint08) [inline src/app/feed/feed-card/feed-card.component.scss]: Added final newline

**Comment by hemantjadon** [comment]
@Achint08 Please mention the things that this PR fixes. I don't see any design changes. I guess the only part done here is linking the twitter button to Twitter?

File: src/app/feed/feed-card/feed-card.component.ts [hunk 3]
```diff
@@ -65,6 +65,12 @@ export class FeedCardComponent implements OnInit {
 		let text = div.textContent || div.innerText || '';
 		return text;
 	}
+    
+private get twitterlink(): string {
```
Comment (hemantjadon) [inline src/app/feed/feed-card/feed-card.component.ts]: I dont think there is any need of this `getter`

File: src/app/feed/feed-card/feed-card.component.html [hunk 4]
```diff
@@ -31,7 +54,7 @@ <h3 class="lightbox-name">{{profileName}}</h3>
 					</span>
 				</a>
 			</div>
-			<div class="card-text">
+			<div class="card-text" (click)="showStyle = !showStyle;">
```
Comment (hemantjadon) [inline src/app/feed/feed-card/feed-card.component.html]: This should not be here instead on complete box. Please try to stay as close to one issue as possible. :)

**Comment by hemantjadon** [comment]
@Achint08 Please mention that what changes have been made in this PR. Like if UI has been changed then before and after screenshots will be good.
If there are other changes please mention that.

**Comment by Achint08** [comment]
Previously the links displayed in the tweets weren't displayed properly and there were no retweet and like options in the lightbox
screen shot 2016-12-24 at 5 46 06 pm

But now the feeds will be displayed properly and options will be displayed in the lightbox

screen shot 2016-12-24 at 5 46 24 pm

**Comment by Achint08** [comment]
Previously the links in the feed weren't displayed properly nor there were retweet and like options in the lightbox
screen shot 2016-12-24 at 5 53 45 pm

Now The feeds will be displayed properly and the options will be displayed as shown.

screen shot 2016-12-24 at 5 54 03 pm

**Comment by hemantjadon** [comment]
Can you quickly squash the commits, so we can merge this.

**Comment by Achint08** [comment]
@hemantjadon  done (Y)

**Comment by hemantjadon** [comment]
Doesn't fix the issue completely. But implements mainly the Autolinking.

**Pull request merged**